### ISSUE TITLE
Missing virtual keyboard in Fuel page of EFB

### ISSUE THREAD

**Issue by Maverickwoe** [flybywiresim/aircraft#8221]
### Aircraft Version

Development
### Build info

```json
{
    "built": "2023-09-16T06:20:39+00:00",
    "ref": "refs/heads/master",
    "sha": "4783ebcadace8462bc71af4f8c1bb4c1ebdb322c",
    "actor": "2hwk",
    "event_name": "manual",
    "pretty_release_name": "master:4783ebca",
    "version": "v0.11.0-dev.4783ebc"
}
```

### Describe the bug

When the virtual keyboard is switched on in the EFB settings, it won't appear on the screen when selecting the fuel quantity in the Fuel page of the EFB even though the keyboard offset is applied.
![image](https://github.com/flybywiresim/docs/assets/12175224/44b2a41e-d0b6-4d6c-9c93-f4e3d3da42e6)

### Expected behavior

The virtual keyboard shall appear.

### Steps to reproduce

1. Open EFB
2. Configure the Virtual Keyboard so that it is active in the EFB settings
3. Go to the Service page and Fuel tab
4. Select the input field for the refuel

Observe that the input field is covered by the keyboard

### Additional info (optional)
 
This was noted in VATSIM
### Discord Username (optional)

garoomf

**Comment by 2hwk** [comment]
Note: Check payload as well it probably does not work there either ✨🔖

**Comment by LouisVenhoff** [comment]
Is this still a problem or is it fixed?

**Comment by 2hwk** [comment]
No, no fix has been pushed as far as I am aware. We are redoing quite a few things in the EFB at the moment so this is not planned to be solved yet.

**Comment by CLC0609** [comment]
Seems to still be an issue. I can work on this.
